src/viam/examples/modules/complex/proto/buf.lock

```diff
@@ -4,5 +4,5 @@ deps:
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: 28151c0d0a1641bf938a7672c500e01d
-    digest: shake256:49215edf8ef57f7863004539deff8834cfb2195113f0b890dd1f67815d9353e28e668019165b9d872395871eeafcbab3ccfdb2b5f11734d3cca95be9e8d139de
+    commit: b30c5775bfb3485d9da2e87b26590ac9
+    digest: shake256:9d0caaf056949a0e1c883b9849d8a2fa66e22f18a2a48f867d1a8c700aa22abee50ad3ef0d8171637457cadc43c584998bdf3adac55da0f9e4614c72686b057d
```

Comment:
This was changed automatically when compiling. Not positive this is right but didn't want to mess around with it.

Reply:
Looks right to me! Just an update to the googleapis buf repo; I haven't seen any errors updating locally, so I think this change is fine in this PR (or whichever PR gets in first, I think #186 has this change already).

Reply:
Should be gone now after rebasing.